nixos/tests/kexec.nix

```diff
@@ -1,22 +1,46 @@
 # Test whether fast reboots via kexec work.
```

Comment:
_fast_ isn't tested (nitpick)

Reply:
what else should we write?

Reply:
just remove _fast_, that works

```
# Test whether reboots via kexec work.
```

Reply:
It's actually a description of the first part of the test script now, which doesn't have a header comment.